Miguel tuñón 8-929-2198  Henry Alvarado 8-939-1856/lab7arreglos.c

```diff
@@ -0,0 +1,19 @@
+#include<stdio.h>
+void main()
+{
+	int c,i,n[100],j[100];
+	
+	printf("Ingrese la cantidad de numeros\n");
+	scanf("%d",&c);
+	
+	for(i=0;i<c;i++)
+	{
+	    printf("Ingrese los nuemros\n");
+	    scanf("%d",&n[i]);
+	    
+	    j[i]=n[i];

```

Comment:
La asignacion en el otro arreglo, debia realizarse en otro ciclo for igual que la impresion

Reply:
Si inserto 100 valores del arreglo los imprime inmediatamente despues que introduzco un valor, **esto no es correcto**